src/version.js

```diff
@@ -92,6 +96,19 @@ async function resolveVersion({ range, isCanary }) {
     return { version: range, isCanary: true };
   }
 
+  if (range === "latest") {
+    const res = await fetchWithRetries(
+      "https://dl.deno.land/release-latest.txt",
+    );
+    if (res.status !== 200) {
+      throw new Error(
+        "Failed to fetch release version info from dl.deno.land. Please try again later.",
+      );
+    }
+    const version = (await res.text()).trim();
```

Comment:
I wonder if we need to call `version = semver.clean(version);` like the below path. That seems stripping the leading `v`

Reply:
Ah, probably we need to call `semver.clean()` to strip the leading `v` because `v` is prepended later here https://github.com/denoland/setup-deno/blob/main/src/install.js#L24

Reply:
yes, good catch. fixed.